setup.py

```diff
@@ -27,7 +27,7 @@
 # fmt: off
 setup(
     name='bluetooth-mesh',
-    version='0.7.20',
+    version='0.7.21',
```

Comment:
@michallowasrzechonek-silvair shouln't this be at least `0.8.0` (as we break the API in some way)?

Reply:
Technically, it's still a `0.x` ;) But yeah, it would be a bit nicer.